docs/prerequisites.md

```diff
@@ -48,7 +48,9 @@ AWS Cloud9 is a cloud-based integrated development environment (IDE) that lets y
 
 5. Name it "workshop", and go to the Next step.
 
-6. Select Create a new instance for environment (EC2) and click _Other instance type_ and pick "t2.medium"
+6. Select Create a new instance for environment (EC2) and click _Other instance type_ and pick "t3.medium"
+
+7. Select **Amazon Linux 2** if it is not already selected
```

Comment:
This is the new default, and brings in the root-is-now-XFS change, which also means the resize.sh has to change a bit